libs/utils/src/file_reading.ts

```diff
@@ -1,43 +1,6 @@
 import JsZip, { JSZipObject } from 'jszip';
 import { Buffer } from 'node:buffer';
 
-export function readFileAsyncAsString(file: File): Promise<string> {
```

Comment:
These file reading utilities were unused, but we can keep them if we think they'll be useful in the future